### PR TITLE
Updated migration guide

### PR DESCRIPTION
**Pull request by johnwight**
This revision includes requested content changes. I have also provided additional json examples of smaller, multiple report requests. I am grateful for any attention given to these to verify that they are correct. 

Please review @nateprice7  @mike2184  @gorrepat @stomko

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by johnwight** [comment]
Please review this PR @ashokgorrepati.

File: migration-guide.md [hunk 1]
```diff
@@ -42,25 +60,114 @@ For example, consider the following 1.4 request:
...
-3. For each of those values, request a breakdown of the top 100 `page` values
-4. Finally, a separate report for each metric. This means you go through steps 1-4 for `pageviews`, then again for `visits`, and so on.
+2. For each of the values returned, request a breakdown of the top 100 `geocity` values.
+3. For each of those values, request a breakdown of the top 100 `page` values.
+4. Request a separate report for each metric. This means you go through steps 1-4 for `pageviews`, then again for `visits`, and so on.
+
+#### First request
+
+```json
+{
```
Comment (mike2184) [inline migration-guide.md]: The following 4 examples are all using the 1.4 report format. These need to be converted to use the 2.0 version.

Comment (johnwight) [reply migration-guide.md]: okay. thanks. I will remove them.

Comment (mike2184) [reply migration-guide.md]: The following guide has examples of doing multiple breakdowns with the 2.0 reporting API format: https://github.com/AdobeDocs/analytics-2.0-apis/blob/master/reporting-multiple-breakdowns.md

**Pull request merged**